.github/workflows/create-release.yml

```
@@ -99,19 +99,21 @@ jobs:
           docker run -d --entrypoint tail --name temp-container-joystream-node $IMAGE-amd64 -f /dev/null
 
           RESULT=$(docker exec temp-container-joystream-node b2sum -l 256 runtime.compact.wasm | awk '{print $1}')
+          TAG_HASH=$(docker exec temp-container-joystream-node /joystream/node --version | awk '{print $2}' | cut -d- -f -2)
```

```suggestion
          VERSION_AND_COMMIT=$(docker exec temp-container-joystream-node /joystream/node --version | awk '{print $2}' | cut -d- -f -2)
```

TAG_HASH is not the best name, VERSION_AND_COMMIT makes more sense

---

Done